pkgs/top-level/all-packages.nix

```diff
@@ -8587,6 +8587,8 @@ let
 
   mpdscribble = callPackage ../tools/misc/mpdscribble { };
 
+  microHttpd = callPackage ../servers/http/micro-httpd { };
```

Comment:
Fair enough, thanks. I just pattern matched on the mini one right below :smile: 

The author actually styles it as `micro_httpd`. My understanding is that we're loath to add underscorey attribute names in general, right?